src/main/java/org/minions/devfund/lourdes/Actor.java

```diff
@@ -0,0 +1,86 @@
+package org.minions.devfund.lourdes;
+
+import java.util.ArrayList;
+
+/**
+ * Class that define the Actor entity.
+ */
+public class Actor {
+    private String name;
+    private ArrayList<Movie> movies;
+
+    /**
+     * Constructor that recieves the actor name.
+     *
+     * @param name represents the actor name.
+     */
+    public Actor(final String name) {
+        this.name = name;
+        this.movies = new ArrayList<Movie>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)